### ISSUE TITLE
Fix high-severity accessibility issues

### ISSUE THREAD

**Issue by gingi** [Azure/BatchExplorer#2287]
Several elements are missing correct ARIA attributes and keyboard navigation attributes, preventing them from being handled correctly by accessible technologies (ATs).
